d2l-dom.js

```diff
@@ -1,17 +1,17 @@
-import * as dom from '@brightspace-ui/core/helpers/dom.js';
+import * as core from '@brightspace-ui/core/helpers/dom.js';
 
 window.D2L = window.D2L || {};
 
 /** @polymerBehavior */
 D2L.Dom = {
 
-	findComposedAncestor: dom.findComposedAncestor,
+	findComposedAncestor: core.findComposedAncestor,
 
-	getComposedChildren: dom.getComposedChildren,
+	getComposedChildren: core.getComposedChildren,
 
-	getComposedParent: dom.getComposedParent,
+	getComposedParent: core.getComposedParent,
 
-	isComposedAncestor: dom.isComposedAncestor,
+	isComposedAncestor: core.isComposedAncestor,
 
 	getOffsetParent: function(node) {
```

Comment:
Was `getOffsetParent` just not needed in `core` yet, so we'll only move stuff as we need it? (makes sense, just curious)

Reply:
So far it's not needed, but from what I know of it, we'll probably want it moved over.  I think it was added here after the original helpers were copied.  I'd considered moving it in my previous PR when I moved the tests, but then realized the that there's a lot of tests to move, and  didn't want it to hold up my effort of getting the focus helps moved.  Really UI just wanted to eliminate the duplicate code, and have the tests and the impl live together for each helper.  I could move this now, or in another PR?

Reply:
Yeah I'm good with whatever. I think it would be nice to eventually just archive this entire repo and have people not need to know about it... the name has become kind of incorrect. So at that point obviously all the things in it would need to be in core. But that doesn't need to happen now.